packages/slice-machine/server/src/api/slices/push.ts

```diff
@@ -45,6 +45,8 @@ export async function pushSlice(
       variations,
     };
 
+    IO.Slice.writeSlice(modelPath, modelWithImageUrl);
```

Comment:
what is this for? why do we write to the file system during a slice push?

Reply:
So the imageUrl is saved, which was one of the causes of the issue :)
tbh, now that the imageUrl is saved and the compareVersion function looks at imageUrls, the hash doesn't really need to be passed to the front-end.

It might be posable to refactor that part out, I'll have to check.